### ISSUE TITLE
cut and paste problem in TLJH release 0.1 guide

### ISSUE THREAD

**Issue by burkett123** [jupyterhub/the-littlest-jupyterhub#377]
The following cut and paste section contains extra characters in several of the places it's used.  Before "master/boot" if your viewer hides the details.

#!/bin/bash
curl https://raw.githubusercontent.com/jupyterhub/the-littlest-jupyterhub/
˓→master/bootstrap/bootstrap.py \
| sudo python3 - \
--admin <admin-user-name>

**Comment by GeorgianaElena** [comment]
Hi @burkett123! Are you referring to the red arrow in the picture bellow (from the pdf documentation)?
![arrow_doc](https://user-images.githubusercontent.com/7579677/60099009-7315b300-975f-11e9-95d1-6853f8f5f065.png)

This arrow it's [sphinx'](http://www.sphinx-doc.org/en/master/)s way of wrapping long code lines and it should not be treated as part of the syntax, it's only an indicator that what's after the arrow belongs to the same line.
I agree that when you copy and paste the command from the pdf, you get some extra chars with the arrow.
You might want to also check the [TLJH html documentation](https://the-littlest-jupyterhub.readthedocs.io/en/latest/topic/customizing-installer.html) which allows you to easily copy and paste a command.

**Comment by burkett123** [comment]
Yes, you are correct.  It's the extra characters when used to cut and 
paste that cause the problem.  If this is an artifact of the sphinx, 
then maybe shorter lines with additional \ would be the solution.

   Just a thought and observation.

Thanks.

**Comment by GeorgianaElena** [comment]
Thanks a lot for the suggestion @burkett123, very much appreciated.
What are your thoughts on splitting the url @yuvipanda :) ?
